papaparse.js

```diff
@@ -682,7 +682,10 @@ License: MIT
 				return;
 			}
 
-			this._start += xhr.responseText.length;
+			// The length of the responseText doesn't corresponds to the actual size we requested due to potential encoding
+			// of some characters that has more than 1 byte. As such, better to just increment the start index by the chunk
+			// size if it was given
+			this._start += this._config.chunkSize ? this._config.chunkSize : xhr.responseText.length;
```

Comment:
Personally I think the better solution here is to use the TextEncoder to properly get the correct byte size. But I'm not sure if it is available in all the browsers you support plus that would also mean we would need to parse the content to get the right size. Now sure if all that is worth it so going with this.

Reply:
@pokoli - I guess I can also use `this._start += +xhr.getResponseHeader('Content-Length');` from another bug request though I'm a little hesitant to use headers since it is dependent on the server which may or may not provide the information. Thoughts?

Reply:
I agree that we should avoid using headers if they are optionals. So current implementation seems ok.